build.gradle

```diff
@@ -99,6 +99,9 @@ task bundle(type: Bundle) {
 }
 
 jar {
+    manifest {
+        attributes('Automatic-Module-Name': 'io.nats.jnats')
+    }
```

Comment:
In the articles I found about this subject, the majority of them show this code:

```
jar {
    manifest {
        name = "mylibrary"
        instruction "Automatic-Module-Name", "io.nats.jnats"
    }
}
```

Reply:
I'll admit that I'm not much of a Maven person, so my proposed solution may not be the most correct one, would you like me to edit the PR or do you guys want to deal with this?